util/regtool.py

```diff
@@ -107,15 +106,24 @@ def main():
 
     verbose = args.verbose
 
-    if args.j: format = 'json'
-    elif args.c: format = 'compact'
-    elif args.d: format = 'html'
-    elif args.doc: format = 'doc'
-    elif args.r: format = 'rtl'
-    elif args.s: format = 'dv'
-    elif args.f: format = 'fpv'
-    elif args.cdefines: format = 'cdh'
-    elif args.ctdefines: format = 'cth'
+    if args.j:
+        format = 'json'
+    elif args.c:
+        format = 'compact'
+    elif args.d:
+        format = 'html'
+    elif args.doc:
+        format = 'doc'
+    elif args.r:
+        format = 'rtl'
+    elif args.s:
+        format = 'dv'
+    elif args.f:
+        format = 'fpv'
+    elif args.cdefines:
+        format = 'cdh'
+    elif args.ctdefines:
+        format = 'cth'
```

Comment:
This looks reasonable enough as a formatting change, but I'm not convinced it's right to parse arguments like this. What should happen if I pass `-s -f`? Run in DV mode and just forget about the `-f` parameter? If we can't change the command line arguments to make it impossible for the user to get this wrong, it might be better to do something like this:

```python3
arg_map = {'j': 'json', 'c': 'compact', ..(the rest)..}
format = None
for k, v in arg_map.items():
    if getattr(args, k):
        if format is not None:
            log.error("Oh no! You specified both ??? and ???")
        format = v
if format is None:
    log.error("Oh no! You didn't specify a format")
```

Reply:
Thanks for suggestion. I will revise as you suggested in a following PR.